test_runner/src/main/kotlin/ftl/reports/MatrixResultsReport.kt

```diff
@@ -23,7 +25,7 @@ Example:
 object MatrixResultsReport : IReport {
     override val extension = ".txt"
 
-    private val percentFormat by lazy { DecimalFormat("#0.00") }
+    private val percentFormat by lazy { DecimalFormat("#0.00", DecimalFormatSymbols(Locale.US)) }
```

Comment:
Because defaults on some locales change `.` into `,` which breaks 3 unit tests. That fix is required because in commit 18b72b5d `maven( url = "https://maven-central.storage-download.googleapis.com/repos/central/data/" )` was changed to `jcenter()` I guess.

Reply:
I moved to jcenter since the old URL was causing build failures locally and is non-standard.

I think this code is related to printing the cost in US dollar so having that locale makes sense.